src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
It is redundant to reset margin and padding for all elements.
Please reset just for elements, which really need it.
Just one css property for which usually uses this selector it is box-sizing